src/modules/Channel/components/ChannelUI/index.tsx

```diff
@@ -123,9 +123,13 @@ const ChannelUI: React.FC<ChannelUIProps> = ({
         <div className="sendbird-conversation__footer__typing-indicator">
           {
             renderTypingIndicator?.()
-              || globalStore?.config?.groupChannel?.enableTypingIndicator && (
+            || (
+              globalStore?.config?.groupChannel?.enableTypingIndicator
+              && globalStore?.config?.groupChannel?.typingIndicatorTypes?.has(TypingIndicatorTypes.Text)
+              && (
```

Comment:
Should we add `&& !hasMoreNext` here? @bang9 @HoonBaek

Reply:
We don't need it. it's not in a message list.